policy/modules/system/userdomain.if

```diff
@@ -2532,6 +2532,7 @@ interface(`userdom_manage_user_home_content_files',`
 	')
 
 	manage_files_pattern($1, user_home_t, user_home_t)
+	allow $1 user_home_t:file map;
```

Comment:
There is a `mmap_manage_files_pattern()` now.

Reply:
used macro